### ISSUE TITLE
Microsoft.Graph.Authentication 1.9.2 and PnP.PowerShell 1.9.0 causes authentication failure

### ISSUE THREAD

**Issue by kmosti** [microsoft/Microsoft365DSC#1822]
#### Details of the scenario you tried and the problem that is occurring
Following the whitepaper, the release pipeline failed to authenticate.

Debugging that, I tried:

* Register-PnPManagementShellAccess
* Granting all scopes graph permissions
* Tried the following to debug: Export-M365DSCConfiguration -Components @("SPOTenantSettings") -Credential $Credential -FileName "test";

#### Verbose logs showing the problem
The detailed error thrown is as follows:

Exception             : System.AggregateException: One or more errors occurred. ---> System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Graph.Authentication.Core, Version=1.9.2.0, Culture=neutral,                        PublicKeyToken=31bf3856ad364e35' or one of its dependencies. The system cannot find the file specified.

I found these modules to be installed, suggesting a versioning issue:              
```
Script     1.9.2      Microsoft.Graph.Authentication
Script     1.9.3      Microsoft.Graph.Authentication
```

I had not installed 1.9.3, so I tried removing both and reinstalling M365DSC dependencies.  
This reinstalled 1.9.3.

I removed them again, then updated the PnP.PowerShell dependency to 1.8.0 (suspecting PnP to install 1.9.3 as a dep), and reinstalled.

This solved the issue.

#### Suggested solution to the issue

1. In "C:\Program Files\WindowsPowerShell\Modules\Microsoft365DSC\1.22.309.1\Dependencies\Manifest.psd1", update ModuleName PnP.PowerShell from 1.9.0 to 1.8.0.
2. Run the below commands to uninstall the modules and reinstall dependencies

```powershell
Uninstall-Module PnP.PowerShell -AllVersions -Force -Verbose
Uninstall-Module Microsoft.Graph.Authentication -AllVersions -Force -Verbose
Update-M365DSCDependencies
```
(make sure to restart PoSh before uninstalling these, in case they're in use)

#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
Export-M365DSCConfiguration -Components @("SPOTenantSettings") -Credential $Credential -FileName "test";
```

#### The operating system the target node is running

OsName               : Microsoft Windows Server 2016 Datacenter
OsOperatingSystemSKU : DatacenterServerEdition
OsArchitecture       : 64-bit
WindowsBuildLabEx    : 14393.5006.amd64fre.rs1_release.220301-1704
OsLanguage           : en-US
OsMuiLanguages       : {en-US}

#### Version of the DSC module that was used ('dev' if using current dev branch)
1.22.309.1

**Comment by andikrueger** [comment]
Could you run the following cmdLet to make sure, you have the latest dependencies installed:

```powershell
Update-M365DSCDependencies
Uninstall-M365DSCOutdatedDependencies
```

This will install the latest dependencies for the latest version and remove all previous versions of Microsoft365DSC and the older dependencies.

**Comment by kmosti** [comment]
I have it working now, however I had to downgrade PnP.PowerShell to 1.8 for it to work.

**Comment by andikrueger** [comment]
This should not be necessary. Could you check if the PnP PowerShell Application is properly registered within Azure AD?

**Comment by kmosti** [comment]
Yes it is (I granted all permissions in this tenant, and they are in fact working properly now without me having changed them between the reported issue and it working again), however I will try to recreate the issue and if I cannot, I will close this ticket.

I did not install the 1.9.3 version of the Authentication module reported above, however it might be due to something else? In any case it might be worth mentioning that DSC 1.22.309.1 will not work with Microsoft.Graph.Authentication v. 1.9.3 installed, just in case someone stumbles upon this ticket in the future :)
